hiplot/experiment.py

```diff
@@ -315,6 +315,15 @@ def from_csv(file: Union[Path, str, IO[str]]) -> "Experiment":
         else:
             return Experiment.from_iterable(csv.DictReader(file))
 
+    @staticmethod
+    def from_dataframe(dataframe) -> "Experiment":  # No type hint to avoid having pandas as an additional dependency
```

Comment:
```suggestion
    def from_dataframe(dataframe: Any) -> "Experiment":  # No type hint to avoid having pandas as an additional dependency
```

This should make mypy happy in the CI :)

Reply:
Done